CHANGELOG.md

```diff
@@ -55,15 +58,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - added basic structure for Nexus Tools written in Rust in the form of a trait
 - added a macro that starts a webserver for one or multiple tools, providing all necessary endpoints
 - added a first, dumb version of secret manager
-- added a dev guide that goes through the steps to use CLI to scaffold a boilerplate tool and implement NexusTool trait
+- added a dev guide that goes through the steps to use CLI to scaffold a boilerplate tool and
+  implement NexusTool trait
 
 ### `nexus-sdk`
 
 #### Added
 
 - added Nexus Sui identifiers module
 - added `object_crawler` that parses Sui objects to structs
-- added `test_utils` that handle spinning up Redis or Sui containers for testing, along with some helper functions
+- added `test_utils` that handle spinning up Redis or Sui containers for testing, along with some
+  helper functions
```

Comment:
please revert the wrapping, we want leave the wrapping to the rendering engine in all md files

Reply:
reverted https://github.com/Talus-Network/nexus-sdk/pull/158/commits/39fbb5b00a027df9a6b0e66ffc996dbe339594fd